### PR TITLE
Optimize operator UI

### LINKED ISSUE #2348: [FEAT]: UI for Optimize operator

### PR DESCRIPTION
**Pull request by Jami159**
UI for optimize operator

Resolves #2348

https://github.com/DARPA-ASKEM/terarium/assets/10509899/561f1ba9-d1b4-4d96-ae51-fbcfe69ee9c4

### CODE REVIEW

File: packages/client/hmi-client/src/components/optimize/tera-intervention-policy-group-form.vue [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+			<div>
+				<i
+					class="trash-button pi pi-trash"
+					:style="'cursor: pointer'"
+					@click="emit('delete-self', index)"
+				/>
+			</div>
+		</div>
+		<div class="input-row">
+			<div class="label-and-input">
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/optimize/tera-intervention-policy-group-form.vue:31]: What about `<fieldset>`?
https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_structure_a_web_form#the_fieldset_and_legend_elements

File: packages/client/hmi-client/src/components/optimize/tera-intervention-policy-group-form.vue [hunk 2]
```diff
@@ -0,0 +1,203 @@
+<template>
+	<div class="policy-group" :style="`border-left: 9px solid ${props.config.borderColour}`">
+		<div class="form-header">
+			<div class="header-items">
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/optimize/tera-intervention-policy-group-form.vue:4]: You can use a `<header>` and remove that class.

File: packages/client/hmi-client/src/workflow/ops/model-optimize/tera-model-optimize-node.vue [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<template>
+	<section>
+		<img src="@assets/svg/plants.svg" alt="" draggable="false" />
+		Configure in the side panel
+	</section>
```
Comment (YohannParis) [inline packages/client/hmi-client/src/workflow/ops/model-optimize/tera-model-optimize-node.vue:5]: We have a [tera-operator-placeholder-graphic](https://github.com/DARPA-ASKEM/terarium/blob/main/packages/client/hmi-client/src/workflow/operator/tera-operator-placeholder-graphic.vue) for that purpose.

And we do not have a side panel anymore, we open the drilldown.

File: packages/client/hmi-client/src/components/optimize/tera-intervention-policy-group-form.vue (hunk 1 above)
Comment (Jami159) [reply packages/client/hmi-client/src/components/optimize/tera-intervention-policy-group-form.vue:31]: I tried this, but this added its own styling that messed up how the form looked :/

File: packages/client/hmi-client/src/components/optimize/tera-intervention-policy-group-form.vue [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+}
+
+.input-row > *:first-child {
+	flex: 2;
+}
+
+.input-row > *:nth-child(2),
+.input-row > *:nth-child(3) {
+	flex: 1;
+}
```
[the remark this answers is not in the document]
Comment (Jami159) [reply packages/client/hmi-client/src/components/optimize/tera-intervention-policy-group-form.vue]: done!

**Review by YohannParis — APPROVED** [review]

**Pull request merged**